tests/unit/src/test/scala/tests/OverridesSuite.scala

```diff
@@ -0,0 +1,152 @@
+package tests
+
+import scala.meta.Input
+
+import com.sourcegraph.lsif_semanticdb.Symtab
+import munit.FunSuite
+import munit.TestOptions
+
+class OverridesSuite extends FunSuite with TempDirectories {
+
+  val targetroot = new DirectoryFixture()
+
+  override def munitFixtures: Seq[Fixture[_]] =
+    super.munitFixtures ++ List(targetroot)
+
+  def checkOverrides(
+      options: TestOptions,
+      source: String,
+      extractSymbol: String,
+      expectedSymbols: List[String],
+      expectedCount: Int,
+      qualifiedClassName: String = "example.Parent"
+  ): Unit = {
+    test(options) {
+      val compiler = new TestCompiler(targetroot())
+      val relativePath = qualifiedClassName.replace('.', '/') + ".java"
+      val input = Input.VirtualFile(relativePath, source)
+      val result = compiler.compileSemanticdb(List(input))
+      val symtab = new Symtab(result.textDocument)
+
+      val count = symtab.symbols.get(extractSymbol).getOverriddenSymbolsCount
+      val syms = symtab
+        .symbols
+        .get(extractSymbol)
+        .getOverriddenSymbolsList
+        .toArray(new Array[String](count))
+      assertEquals(count, expectedCount)
```

Comment:
We can also remove the `zipWithIndez.foreach` assertions
```suggestion
      assertEquals(syms, expected)
```